dbms/src/Functions/timezone.cpp

```diff
@@ -1,3 +1,4 @@
+#include <Functions/registerFunctions.h>
```

Comment:
Not sure this is not an overkill. But let's proceed...

Reply:
It is MEGA overkill
I'm already afraid of this PR :)